client-mvp-04/src/utils/uiKit/uiKit.js

```diff
@@ -162,6 +162,28 @@ const DevUiKit = () => {
       </div>
       <p className={'dev-comment'}>Input with icon</p>
 
+      <label for="big-radio-button" className={'radio-button-container'}>
```

Comment:
Attribute for is not allowed here
For react, attribute `for` becomes `htmlFor`
https://reactjs.org/docs/dom-elements.html#htmlfor